payrexx/src/Service/PayrexxOrderService.php

```diff
@@ -98,16 +102,33 @@ public function getPrestaStatusByPayrexxStatus($transactionStatus)
      */
     public function transitionAllowed($newStatus, $oldStatusId)
     {
+        $refundStatusId = (int) \Configuration::get(self::PS_STATUS_REFUND);
+        $newStatusId = (int) \Configuration::get($newStatus);
+        if ($oldStatusId === $newStatusId && $newStatusId !== $refundStatusId) {
+            return false;
+        }
+        $orderFinalStatuses = [
+            (int) \Configuration::get(self::PS_STATUS_REFUND),
+            (int) \Configuration::get(self::PS_STATUS_PAYMENT),
+            (int) \Configuration::get(self::PS_STATUS_SHIPPING),
+            (int) \Configuration::get(self::PS_STATUS_DELIVERED),
```

Comment:
Status Bankwire is missing. Or is removed on purpose?

Reply:
@mathis-hermann 

The PS_OS_BANKWIRE(10) text is Awaiting bank wire payment

It is not a final state.

Reply:
@vinothss4u Ok - but then this case ` return $oldStatusId !== (int) \Configuration::get(self::PS_STATUS_BANKWIRE);` (old line 109) is missing?